### PR TITLE
Change style of Button Template

### PR DESCRIPTION
**Pull request by mastasky**
Colour button template top in theme primary color
Line breaks in Button Template top

### CODE REVIEW

File: src/plugins/messenger/MessengerPreview/components/MessengerButtonTop.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { MessagePluginFactoryProps } from '../../../../common/interfaces/message-plugin';
+
+export const getMessengerButtonTop = ({ React, styled }: MessagePluginFactoryProps) => {
+    const MessengerFrame = styled.div(({ theme }) => ({
+        width: 250,
+        borderRadius: theme.unitSize * 2,
+        overflow: 'hidden',
+        background: theme.primaryGradient,
+        color: theme.primaryContrastColor,
```
Comment (dshire) [inline src/plugins/messenger/MessengerPreview/components/MessengerButtonTop.tsx:9]: Looks nice with the theming, but why not add it everywhere then (lists, galleries)? Also, we should probably make this an optional config setting as it changes previous behavior

**Review by dshire — CHANGES_REQUESTED** [review]

File: src/plugins/messenger/MessengerPreview/components/MessengerButtonTop.tsx (hunk 1 above)
Comment (mastasky) [reply src/plugins/messenger/MessengerPreview/components/MessengerButtonTop.tsx:9]: Lists and Galleries are different Buttons are more like QRs. I consider it a bug before. Let's keep only here.

**Comment by mastasky** [comment]
@pedily  - please make sure that linebreaks can be turned off with setting

**Review by dshire — APPROVED** [review]

File: src/plugins/messenger/MessengerPreview/components/MessengerButtonTemplate/MessengerButtonTemplate.tsx [hunk 2]
```diff
@@ -13,12 +13,13 @@ export const getMessengerButtonTemplate = ({
...
 }: MessagePluginFactoryProps) => {
-    const MessengerFrame = getMessengerFrame({ React, styled });
+    const MessengerButtonHeader = getMessengerButtonHeader({ React, styled });
     const MessengerButton = getMessengerButton({ React, styled });
     const Divider = getDivider({ React, styled });
 
     const Text = styled.div(({ theme }) => ({
-        padding: theme.unitSize
+        padding: `${theme.unitSize * 2}px ${theme.unitSize * 3}px`,
+        whiteSpace: "pre-line"
```
Comment (mastasky) [inline src/plugins/messenger/MessengerPreview/components/MessengerButtonTemplate/MessengerButtonTemplate.tsx:22]: @pedily Please make this optional via config

**Review by kwinto — APPROVED** [review]
lgtm

**Pull request merged**